### DISCUSSION TITLE
no-data when reprojecting ETRS89 / UTM zone 32N to WGS84!

### DISCUSSION THREAD

**Discussion by PeterFogh** [corteva/rioxarray#435] [Q&A] [answered]
Hi, I'm working on the public available Danish Elevation Model here https://github.com/PeterFogh/digital_elevation_model_use_cases.
However, when I try to reproject the GeoTIFF file in ETRS89 / UTM zone 32N to WGS84 the resulting dataset is filled with no-data values. Does anybody know why this is happening for my dataset?

I have tried executing with documentation data (see [docs](https://corteva.github.io/rioxarray/stable/examples/reproject.html )) with my package version without any problems.

Here is one of my GeoTIFF files from the Danish Elevation Model which fails.
[DTM_1km_6280_540.zip](https://github.com/corteva/rioxarray/files/7574930/DTM_1km_6280_540.zip)

OS version: Ubuntu 20.04.3 LTS
package versions: 
```
rioxarray.__version__='0.8.0'
xr.__version__='0.19.0'
rasterio.__version__='1.2.10'
```

My code and package version are:
```
import rioxarray
import xarray as xr
import rasterio
print(f'{rioxarray.__version__=}')
print(f'{xr.__version__=}')
print(f'{rasterio.__version__=}')

da = rioxarray.open_rasterio('../DEMs/denmark/data/DTM_1km_6285_540.tif')
da.rio.reproject('epsg:4258').plot()
```
Which results in this visualization

![Screenshot from 2021-11-20 18-39-20](https://user-images.githubusercontent.com/5435299/142736020-7422022b-6ee3-46a4-b6b6-f03bcb99e7eb.png)

**Comment by snowman2** [comment, marked as answer]
What happens if you mask the nodata:
```python
da = rioxarray.open_rasterio('../DEMs/denmark/data/DTM_1km_6285_540.tif')
dar = da.rio.reproject('epsg:4258')
dar.where(dar!=dar.rio.nodata).plot()
```

**Comment by PeterFogh** [comment]
Hi @snowman2, thank you so much :) it solved the problem.

Thus the problem was, that the plotting could not visualise the actual data when the nodata value is so large (-9999) (e.g. the range of values was too big to show the range of the actual data)

![Screenshot from 2021-11-22 08-32-29](https://user-images.githubusercontent.com/5435299/142821171-e69747ef-5dc9-4f2c-9ddc-15612da12145.png)
